bedrock/settings/__init__.py

```diff
@@ -94,7 +90,38 @@
 STATIC_URL = CDN_BASE_URL + STATIC_URL
 logging.config.dictConfig(LOGGING)
 
+# OPERATION MODE SELECTION
+# Which site do we want Bedrock to serve?
+DEFAULT_SITE_MODE = "Default"
+POCKET_SITE_MODE = "Pocket"
+MOZORG_SITE_MODE = "Mozorg"
+
+site_mode = config("SITE_MODE", default=DEFAULT_SITE_MODE)
+
+if site_mode == MOZORG_SITE_MODE:
+    ROOT_URLCONF = "bedrock.urls_mozorg_mode"
+elif site_mode == POCKET_SITE_MODE:
+    ROOT_URLCONF = "bedrock.urls_pocket_mode"
+    # TODO: from settings.pocket import *
+    # to switch in Pocket-appropriate versions of
+    # DEV_LANGUAGES, PROD_LANGUAGES, CANONICAL_LOCALES,
+    # FLUENT_* overrides for Pocket L10N, etc
+    # (The import from settings.pocket is just one option)
```

Comment:
This is as much a talking point as a note to myself: we could inline-redefine pocket-specific settings here, or pull them in from _another_ file. Given we're so far retaining the bedrock/settings/* module anyway, it's not the end of the world, but I'm open to opinions